bbc/simorgh
# Update styles.js to use relative units

Transitions styles.js from using pixels to relative units (`rem`s) for spacing only. See 407 for the spike. `rem`s are chosen on the basis of clarity and no perceptible difference in rendering.


* [x] Fixes #407
* [x] Tests added for new features (snapshots updated)


**Screenshots**
***Figcaption***
![rems - figcaption](https://user-images.githubusercontent.com/916416/43950253-b36a49bc-9c87-11e8-9274-ee5667e5d5a1.png)

***Footer***
![rems - footer](https://user-images.githubusercontent.com/916416/43950255-b38c4472-9c87-11e8-8592-fedc4e9cce44.png)

![rems - footer2](https://user-images.githubusercontent.com/916416/43950256-b3b41826-9c87-11e8-82b1-c6e342451215.png)

***H1***
![rems - h1](https://user-images.githubusercontent.com/916416/43950258-b3d6a288-9c87-11e8-83e4-058fc90acc36.png)

***H2***
![rems - h2](https://user-images.githubusercontent.com/916416/43950259-b3f88646-9c87-11e8-8f3d-1231f9dd8889.png)


**Testing instructions**
Headings, figcaption and footer have had their spacing set using REMs, so testing should check the branch has the same computed value in pixels as `latest`.

1. Set browser font-size to 16px. Check that the following computed padding values are true
* h1 should have padding-top of 32px and padding-bottom of 16px
* h2 should have padding-top and padding-bottom of 16px
* figcaption should have padding of 8px
* footer: 
** "Text here. Link last" p should have padding-top and padding-bottom of 16px. Inner a should have padding-top and padding-bottom of 8px
** both ul should have padding-top and padding-bottom of 8px
** first li should have padding-bottom and margin-bottom of 8px
** other li > a should have padding-top and padding-bottom of 8px

2. Change browser font-size to 20px, which changes the value of 1 rem to equal 20px. 
Refresh and inspect the h1. Check that the h1's padding-top is 40px and that its padding-bottom is 20px
